crates/telio-model/src/api_config.rs

```diff
@@ -113,13 +115,28 @@ pub struct FeatureNurse {
 }
 
 impl FeatureNurse {
+    /// One hour
+    pub const DEFAULT_HEARTBEAT_INTERVAL: u32 = 3600;
```

Comment:
Can you make it `Duration` and use `.as_secs()` downstairs?

Reply:
I would leave it as u32 as the field in FeatureNurse is u32

Reply:
But u32 3600 doesn't mean anything. A person looking for the default value might assume it's minutes or something

Reply:
I understand that, but every Duration in api_config is like that